src/utils/tools/get-lab-config.sh

```diff
@@ -1,38 +1,62 @@
 #!/usr/bin/env bash
 
-if [[ "${USER_LAB}" == "" || "${TOKEN_LAB}" == "" ]]; then
-    echo "USER_LAB and/or TOKEN_LAB environment variables not set."
+# Check if both parameters are provided as arguments
+if [ $# -eq 2 ]; then
+    CLUSTER_NUMBER=$1
```

Comment:
Maybe check if $1 is indeed a number

Reply:
not sure if we need to overcomplicate it